spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -186,6 +186,18 @@
     expect(vm.host).to eq(vm2.host)
   end
 
+  xit 'handles refresh of new target without deleting other inventory' do
```

Comment:
I guess this is disabled because it would need https://github.com/ManageIQ/manageiq/pull/14247 ?